src/components/webcam-texture-target.js

```diff
@@ -0,0 +1,84 @@
+import { disposeTexture } from "../utils/material-utils";
+
+AFRAME.registerComponent("webcam-texture-target", {
+  schema: {
+    src: { type: "string" }
+  },
+
+  getMaterial() {
```

Comment:
Hmm, seems like we don't actually end up taking into account which material to replace based on what materials had the component on it. 

I would say we either want to account for that (probably by keeping track of it somehow at inflation time), or just have this component be something you add to a Mesh instead, and then keep the current behavior of just swapping that mesh's material (getting rid of the extra code around inflating material components).

Reply:
In this design, the component is not responsible for where it is attached. `gltf-model-plus` was modified such that material components are inflated and placed on an entity that is the direct parent of the mesh object.